Source/WTF/wtf/unix/LoggingUnix.cpp

```diff
@@ -29,11 +29,34 @@
 #include <wtf/text/MakeString.h>
 #include <wtf/text/WTFString.h>
 
+#if OS(ANDROID)
+#include <sys/system_properties.h>
+
+static const char* androidLogLevelString()
+{
+    const auto* propertyInfo = __system_property_find("debug.wpewebkit.log");
```

Comment:
Maybe remove `wpe` here?

Reply:
I'm OK with both approaches. 
- `debug.webkit.log` is consistent with `WEBKIT_DEBUG` as we normally use it, 
- `debug.wpewebkit.log` is possibly fairer considering we're only having active Android efforts for the WPEWebKit port

Reply:
My reason to use `debug.wpewebkit.log` here was that [people have been exploring how to run WebKitGTK on Android](https://gitlab.gnome.org/GNOME/glib/-/issues/3449#note_2473318) (no success yet), so at some point we may want to have also `debug.webkitgtk.log` as well&mdash;we may want to allow configuring logging separately for each port, in case the GTK port is eventually made to work on Android.

Thinking about that, probably the best approach could be using something like this:

```suggestion
    const auto* propertyInfo =
        __system_property_find("debug." LOG_CHANNEL_WEBKIT_SUBSYSTEM ".log");
```

This would result in `debug.WPEWebKit.log` for the WPE port, and `debug.WebKitGTK.log` for the GTK one. Also, this makes the name match the “log tag” passed to `__android_log_{print,vprint,write}` in https://github.com/WebKit/WebKit/pull/47339, making everything a bit more consistent.

Thoughts?

Reply:
I think this is a good idea!

Reply:
I have updated the patch in this way. PTAL.